Automattic/wp-calypso
# Google Photos: update copy

#### Changes proposed in this Pull Request

Update all the Photos from Google copy to Google Photos. Also updates icons with the official Google Photos logo.

Changes:

1. Sharing page has a new logo, updated text, and a new screenshot:
<img width="744" alt="modified" src="https://user-images.githubusercontent.com/1277682/54017719-ca771f80-417e-11e9-8793-f203fee046ef.png">

2. Media source dropdown has a new logo (both in the menu and in the selected position), as well as updated text:

<img width="245" alt="dropdown" src="https://user-images.githubusercontent.com/1277682/54017784-fc888180-417e-11e9-9d78-c6d3cc2c5ba4.png">

3. When viewing the media library and not currently connected there is a new logo and updated text:
 
<img width="521" alt="connect" src="https://user-images.githubusercontent.com/1277682/54017817-1a55e680-417f-11e9-8d36-ecc80f26270b.png">

Note that `google-photos-icon.jsx` is added as a custom SVG React component so it can fit in with the rest of the SVG behaviour provided by Gridicons. While not ideal, given the deadline and upcoming media library rewrite this seems acceptable.

### Testing

You will need to apply the API patch in `D23306-code` to test this, and `public-api.wordpress.com` needs to be sandboxed.

1. Visit the Sharing page and disconnect from the existing Google Photos
2. Verify that the Google Photos display matches screenshot (1)
3. Visit the Media page and click source dropdown in top left corner. Verify that the dropdown matches screenshot (2)
4. Select Google Photos. Verify that the dropdown icon correctly shows hover states to match other items
5. Verify that screenshot (3) is shown in the media library
